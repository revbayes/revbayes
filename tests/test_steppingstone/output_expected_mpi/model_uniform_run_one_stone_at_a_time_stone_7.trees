Iteration	Posterior	Likelihood	Prior	psi
0	-24068.43	-24058.14	-10.28619	((((Chlorocebus_aethiops[&index=5]:0.074152,Macaca_mulatta[&index=13]:0.194103)[&index=25]:0.097043,Colobus_guereza[&index=6]:0.073564)[&index=32]:0.024758,(Pan_paniscus[&index=17]:0.061464,(((((Cebus_albifrons[&index=3]:0.008645,Saimiri_sciureus[&index=20]:0.196675)[&index=43]:0.410425,(Daubentonia_madagascariensis[&index=7]:0.358226,Aotus_trivirgatus[&index=1]:0.046822)[&index=26]:0.013407)[&index=41]:0.028180,Callicebus_donacophilus[&index=2]:0.009238)[&index=33]:0.043261,Loris_tardigradus[&index=12]:0.255098)[&index=36]:0.072838,Hylobates_lar[&index=9]:0.283725)[&index=28]:0.009280)[&index=30]:0.302327)[&index=42]:0.070580,(Propithecus_coquereli[&index=19]:0.172763,((((Lepilemur_hubbardorum[&index=11]:0.371651,(Varecia_variegata_variegata[&index=22]:0.286139,(Tarsius_syrichta[&index=21]:0.056096,(Cheirogaleus_major[&index=4]:0.125096,Lemur_catta[&index=10]:0.019404)[&index=35]:0.038643)[&index=37]:0.079039)[&index=40]:0.018201)[&index=31]:0.177184,((Otolemur_crassicaudatus[&index=16]:0.350277,Galago_senegalensis[&index=8]:0.371697)[&index=24]:0.009991,Perodicticus_potto[&index=18]:0.058104)[&index=34]:0.128164)[&index=39]:0.020241,Microcebus_murinus[&index=14]:0.122642)[&index=29]:0.088555,Nycticebus_coucang[&index=15]:0.053069)[&index=38]:0.026237)[&index=27]:0.186030,Galeopterus_variegatus[&index=23]:0.307591)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
10	-25119.05	-25120.8	1.749964	(Galeopterus_variegatus[&index=23]:0.100734,((((Lepilemur_hubbardorum[&index=11]:0.169376,(Varecia_variegata_variegata[&index=22]:0.269505,Otolemur_crassicaudatus[&index=16]:0.155466)[&index=24]:0.010993)[&index=29]:0.063495,Propithecus_coquereli[&index=19]:0.086722)[&index=40]:0.009459,((Lemur_catta[&index=10]:0.022948,Microcebus_murinus[&index=14]:0.095936)[&index=35]:0.077258,Cheirogaleus_major[&index=4]:0.049935)[&index=31]:0.036445)[&index=27]:0.090867,Nycticebus_coucang[&index=15]:0.048421)[&index=38]:0.041287,((((Aotus_trivirgatus[&index=1]:0.077769,((Saimiri_sciureus[&index=20]:0.192050,(Callicebus_donacophilus[&index=2]:0.007475,Cebus_albifrons[&index=3]:0.041591)[&index=41]:0.012975)[&index=43]:0.240023,Loris_tardigradus[&index=12]:0.143147)[&index=26]:0.023473)[&index=33]:0.053904,(Macaca_mulatta[&index=13]:0.131659,(Hylobates_lar[&index=9]:0.307012,(Perodicticus_potto[&index=18]:0.219308,Galago_senegalensis[&index=8]:0.169948)[&index=34]:0.339024)[&index=25]:0.103716)[&index=39]:0.014948)[&index=28]:0.013276,(Pan_paniscus[&index=17]:0.037405,(Tarsius_syrichta[&index=21]:0.086841,Daubentonia_madagascariensis[&index=7]:0.300621)[&index=37]:0.329863)[&index=36]:0.073157)[&index=42]:0.032909,(Chlorocebus_aethiops[&index=5]:0.034329,Colobus_guereza[&index=6]:0.059126)[&index=32]:0.025697)[&index=30]:0.082483)[&index=44];
