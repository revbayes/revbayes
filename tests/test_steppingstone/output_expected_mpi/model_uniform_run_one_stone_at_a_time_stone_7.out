state	power	likelihood
4	0.00243	-25028.5
5	0.00243	-24434.6
6	0.00243	-25128.6
7	0.00243	-25800.8
8	0.00243	-25747.7
9	0.00243	-25475.5
10	0.00243	-25107.1
